### PR TITLE
CLOUD-706 run eks in parallel

### PR DESCRIPTION
**Pull request by nmarukovich**

### CODE REVIEW

**Comment by tplavcic** [comment]
@nmarukovich Do you have a link to test run in Jenkins with this pipeline?

File: cloud/jenkins/psmdb_operator_eks.groovy [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+        'delete-cluster-after-hours': '10'
+EOF
+    """
+
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'eks-cicd', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+        sh """
+            export KUBECONFIG=/tmp/${CLUSTER_NAME}-${CLUSTER_SUFFIX}
+            export PATH=/home/ec2-user/.local/bin:$PATH
+            source $HOME/google-cloud-sdk/path.bash.inc
+            eksctl create cluster -f cluster-${CLUSTER_SUFFIX}.yaml
```
Comment (cap1984) [inline cloud/jenkins/psmdb_operator_eks.groovy:52]: let's use creators' suggestion

```eksctl create cluster --name=cluster-2 --nodes=4 --kubeconfig=./kubeconfig.cluster-2.yaml```

Comment (nmarukovich) [reply cloud/jenkins/psmdb_operator_eks.groovy:52]: I think to use file is more comfortable. @tplavcic what do you think?

**Comment by nmarukovich** [comment]
> @nmarukovich Do you have a link to test run in Jenkins with this pipeline?

https://cloud.cd.percona.com/view/PSMDB/job/psmdb-operator-eks/

**Review by cap1984 — APPROVED** [review]

File: cloud/jenkins/psmdb_operator_eks_latest.groovy [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+        return 0
+    }
+    runTest(TEST_NAME)
+}
+
+void installRpms() {
+    sh """
+        sudo yum install -y jq | true
+    """
+}
```
Comment (tplavcic) [inline cloud/jenkins/psmdb_operator_eks_latest.groovy:191]: Missing space. Please change the same in other file.

File: cloud/jenkins/psmdb_operator_eks_latest.groovy [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'eks-cicd', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+        sh """
+            export KUBECONFIG=/tmp/$CLUSTER_NAME-${CLUSTER_SUFFIX}
+            eksctl delete addon --name aws-ebs-csi-driver --cluster $CLUSTER_NAME-${CLUSTER_SUFFIX} --region eu-west-3
+            eksctl delete cluster -f cluster-${CLUSTER_SUFFIX}.yaml --wait --force --disable-nodegroup-eviction
+        """
+    }
+}
+
+
```
Comment (tplavcic) [inline cloud/jenkins/psmdb_operator_eks_latest.groovy:67]: Excess space.

File: cloud/jenkins/psmdb_operator_eks_version.groovy [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'eks-cicd', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+        sh """
+            export KUBECONFIG=/tmp/$CLUSTER_NAME-${CLUSTER_SUFFIX}
+            eksctl delete addon --name aws-ebs-csi-driver --cluster $CLUSTER_NAME-${CLUSTER_SUFFIX} --region eu-west-3
+            eksctl delete cluster -f cluster-${CLUSTER_SUFFIX}.yaml --wait --force --disable-nodegroup-eviction
+        """
+    }
+}
+
+
```
Comment (tplavcic) [inline cloud/jenkins/psmdb_operator_eks_version.groovy:67]: Excess space. Please change the same in other file.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: cloud/jenkins/psmdb_operator_eks.groovy (hunk 1 above)
Comment (tplavcic) [reply cloud/jenkins/psmdb_operator_eks.groovy:52]: Generally I prefer command line options, but we use config file here from the start and I'm not sure how complex cli options would look like for all the options in config file - maybe it was the reason to set it up like this from the start.

**Review by tplavcic — APPROVED** [review]

**Pull request merged**